pyproject.toml

```diff
@@ -61,6 +61,8 @@ pytest-cov = "^2.11.1"
 ipython = "^7.22.0"
 
 [tool.poetry.plugins."smqtk_plugins"]
+"xaitk_saliency.impls.vis_sal_classifier.occlusion_scoring" = "xaitk_saliency.impls.vis_sal_classifier.occlusion_scoring"
```

Comment:
@Purg @vbhavank just a question, but is our preference for the key to be `xaitk_saliency.impls.vis_sal_classifier.occlusion_scoring` or `impls.vis_sal_classifier.occlusion_scoring`

Reply:
I assume this is coming from my use of an `"impls...` starting key in my branch. The key needs to be unique just within the package scope (I believe), so since all local impls will be in the same package, I thought it would be shorter to leave off the redundant package name in the config. To that end I think *I* currently prefer just starting with "impl" on the *key* side. The *value* side still needs to be a valid module path, and just *needs* to include the `xaitk_saliency` part.